llama_stack/cli/stack/_build.py

```diff
@@ -88,6 +88,29 @@ def run_stack_build_command(args: argparse.Namespace) -> None:
                 color="red",
             )
             sys.exit(1)
+    elif args.providers:
+        providers = dict()
+        for api_provider in args.providers:
+            api, provider = api_provider.split("=")
```

Comment:
there may not be an `=` in the arg in which case this will error out

Reply:
added a try except value error around this